src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -303,20 +304,10 @@ public BasicStudyInfos createStudy(UUID caseUuid, String userId, UUID studyUuid)
     }
 
     private void createStudyAsync(UUID caseUuid, String userId, BasicStudyInfos basicStudyInfos) {
-        AtomicReference<Long> startTime = new AtomicReference<>();
```

Comment:
Maybe remove the whole method?

Reply:
Should those actions now be made synchronously ?